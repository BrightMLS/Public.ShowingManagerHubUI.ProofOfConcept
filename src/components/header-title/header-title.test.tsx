import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import HeaderTitle from './header-title'

describe('<HeaderTitle /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<HeaderTitle title={'title'} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
