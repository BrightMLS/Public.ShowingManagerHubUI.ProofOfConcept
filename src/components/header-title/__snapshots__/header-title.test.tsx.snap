// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderTitle /> Component should render without crashing 1`] = `
<WithStyles(ForwardRef(Typography))
  className="makeStyles-root-1"
  variant="h1"
>
  title
</WithStyles(ForwardRef(Typography))>
`;
